src/types.ts

```diff
@@ -0,0 +1,24 @@
+// export something to force webpack to see this as an ES module
```

Comment:
lol

Reply:
most likely we can remove these at this point actually so we should be doing that. I'll update